lib/worker.py

```diff
@@ -83,6 +84,7 @@ def get_task_groups():
                 'task_ids': stable_task_ids,
                 'is_parallel': suite.is_parallel(),
                 'fragile_retries': 0,
+                'fragile_checksums': suite.fragile_checksums(),
```

Comment:
As I proposed, just add the "is_fragile" flag.

Reply:
Right, added flag is_fragile.